### PR TITLE
chore: set up code of conduct

### LINKED ISSUE #7: Set up organization wide code of conduct
RDMO uses [Contributor Covenant](http://contributor-covenant.org/), version 1.4.

Ref: https://github.com/rdmorganiser/rdmo/blob/main/CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by afuetterer**
Fixes #7

## Tasks
- [x] update email
- [ ] version 2.1 alright?
- [ ] wording alright? (project maintainers, team vs community leaders)

### CODE REVIEW

**Comment by afuetterer** [comment]
I used Contributor Covenant code of conduct version 2.1 as a basis.

The RDMO code of conduct did not have the headings and replaced community leaders with project maintainers and project team.

Is this still desired? Is RDMO "run" by project maintainers or by the community?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the project team responsible for enforcement at rdmo-team@listserv.dfn.de.
```
Comment (afuetterer) [inline CODE_OF_CONDUCT.md]: Who is getting these e-mails?

Comment (MyPyDavid) [reply CODE_OF_CONDUCT.md]: `rdmo-team@`, I don't know, maybe @jochenklar knows?

Comment (jochenklar) [reply CODE_OF_CONDUCT.md]: Right ... this is outdated. It should be rdmo-contact@listserv.dfn.de now.

Comment (afuetterer) [reply CODE_OF_CONDUCT.md]: Alright, will fix that.

**Comment by afuetterer** [comment]
If the email is updated, is the rest alright?

This is Contributor Covenant code of conduct version 2.1, instead of 1.4.

Are there "community leaders" or is the wording of the rdmo contributor guide, with project team and maintainers still applicable?

**Comment by jochenklar** [comment]
I think I should check the wording with the Steuerungsgruppe. I will have a look tomorrow.

**Comment by afuetterer** [comment]
> I think I should check the wording with the Steuerungsgruppe. I will have a look tomorrow.

Excellent.

**Comment by jochenklar** [comment]
Thanks, for the work, but I think the changes are to big to be just merged with a PR. This needs a bit more discussion. I actually like the old one better, but I am open for discussion. (For my taste the "Enforcements Guidelines" is to verbose and should not be in the CoC.)

Some wording needs to be clarified, as you mention "Project maintainers" and "Project team" do not really apply. We are in the process to create a Verein for RDMO so I think this needs to be reflected in the CoC somehow.

Another question is if we need a CoC for just the repositories, or maybe better for the whole community.

I put in on the agenda for the next software meeting.

**Comment by afuetterer** [comment]
> Thanks, for the work, but I think the changes are to big to be just merged with a PR. This needs a bit more discussion. I actually like the old one better, but I am open for discussion. (For my taste the "Enforcements Guidelines" is to verbose and should not be in the CoC.)

Sure, no problem. Take your time. We can always just stick with the version in rdmo (contributor guide 1.4) and put it here.

> Some wording needs to be clarified, as you mention "Project maintainers" and "Project team" do not really apply. We are in the process to create a Verein for RDMO so I think this needs to be reflected in the CoC somehow.

I see.
 
> Another question is if we need a CoC for just the repositories, or maybe better for the whole community.

In my opinion, this should be the central place for the CoC for the whole community, i.e. have the wording etc. fixed, put it here. And remove the one from the rdmo repository (same goes for issue and PR templates). GitHub will use the "community health files" from the central `.github` in the organization.
 
> I put in on the agenda for the next software meeting.

Nice, I will join the next one.

**Comment by afuetterer** [comment]
Did you have to change to talk about this?

I can also just take this text: https://github.com/rdmorganiser/rdmo/blob/main/CODE_OF_CONDUCT.md

**Pull request closed without merging**